pkg/kubeclient/banneduser.go

```diff
@@ -21,17 +21,17 @@ type bannedUserClient struct {
 }
 
 type BannedUserInterface interface {
-	List(email string) (*crtapi.BannedUserList, error)
+	ListByHashedLabel(value, label string) (*crtapi.BannedUserList, error)
 }
 
-// List returns a BannedUserList containing any BannedUser resources that have a label matching the specified email address
-func (c *bannedUserClient) List(email string) (*crtapi.BannedUserList, error) {
+// ListByHashedLabel returns a BannedUserList containing any BannedUser resources that have a label matching the specified label
```

Comment:
What I actually meant is something like this:

```
type BannedUserInterface interface {
	ListByEmail(email string) (*crtapi.BannedUserList, error)
        ListByPhone(phone string) (*crtapi.BannedUserList, error)
}
...
func (c *bannedUserClient) ListByEmail(email string) (*crtapi.BannedUserList, error) {
     return c.listByHashedLabel("<email_label_key>", email)
}
func (c *bannedUserClient) ListByPhone(phone string) (*crtapi.BannedUserList, error) {
     return c.listByHashedLabel("<phone_label_key>", phone)
}
func (c *bannedUserClient) listByHashedLabel(labelKey, labelValue string) (*crtapi.BannedUserList, error) {
     ...
}
```

Reply:
fixed as part of these commits: 956c1f0 and 7a21751